### PR TITLE
Add reviews tab and show more info in panels

### PR DESCRIPTION
**Pull request by JsonLitz**
- Create separate tab for reviews
- Add status information to panels 

https://trello.com/c/idNCnjFA
https://trello.com/c/hiaBdfJT
screen shot 2017-08-21 at 12 10 20 pm
screen shot 2017-08-21 at 12 09 54 pm

### CODE REVIEW

File: client/src/components/Listing.jsx [hunk 1]
```diff
@@ -154,6 +154,9 @@ class Listing extends React.Component {
       }
     }
 
+    thumbClick (e) {
+      console.log('thumclicked');
```
Comment (clarencebowen) [inline client/src/components/Listing.jsx]: What's supposed to happen when this is clicked?

File: client/src/components/Main.jsx [hunk 2]
```diff
@@ -33,6 +34,12 @@ var Main = (props) => (
 	        userId={props.userId}
 	        socket={props.socket}/>
 	      )}/>
+			<Route exact path="/testimonials" render={(propz) => (
+					<Testimonials
+						{...propz}
```
Comment (clarencebowen) [inline client/src/components/Main.jsx:39]: Typo on "props"

File: client/src/components/Listing.jsx (hunk 1 above)
Comment (JsonLitz) [reply client/src/components/Listing.jsx]: removed

File: client/src/components/Main.jsx (hunk 2 above)
Comment (clarencebowen) [reply client/src/components/Main.jsx:39]: Nvm

**Comment by clarencebowen** [comment]
Go ahead and merge.

**Pull request merged**